### ISSUE TITLE
Error after updating dependencies

### ISSUE THREAD

**Issue by SilencerWeb** [react-community/create-react-native-app#717]
Erorr:
![image](https://user-images.githubusercontent.com/20713191/43136327-c9f58ef4-8f61-11e8-819f-9494bb26afd1.png)

Updated packages:
![image](https://user-images.githubusercontent.com/20713191/43136359-e88280d4-8f61-11e8-80bc-b7dfaa3b8ab1.png)

SDK version is updated to 28.0.0 in app.json as mentioned [here](https://github.com/react-community/create-react-native-app/blob/master/react-native-scripts/template/README.md#updating-to-new-releases)

full package.json:
```js
{
  "name": "deadline-tracker",
  "version": "1.0.0",
  "dependencies": {
    "apollo-cache-inmemory": "^1.2.5",
    "apollo-client": "^2.3.5",
    "apollo-link": "^1.2.2",
    "apollo-link-http": "^1.5.4",
    "apollo-link-ws": "^1.0.8",
    "apollo-upload-client": "^8.1.0",
    "apollo-utilities": "^1.0.16",
    "expo": "^28.0.0",
    "graphql": "^0.13.2",
    "graphql-tag": "^2.9.2",
    "react": "16.4.1",
    "react-apollo": "^2.1.9",
    "react-native": "~0.56.0",
    "react-native-status-bar-height": "^2.0.0",
    "react-native-svg-uri": "^1.2.3",
    "react-native-swipeout": "^2.3.4",
    "react-router": "^4.3.1",
    "react-router-native": "^4.3.0",
    "styled-components": "^3.3.3",
    "subscriptions-transport-ws": "^0.9.12"
  },
  "scripts": {
    "start": "react-native-scripts start",
    "eject": "react-native-scripts eject",
    "android": "react-native-scripts android",
    "ios": "react-native-scripts ios",
    "test": "jest",
    "precommit": "lint-staged"
  },
  "devDependencies": {
    "babel-eslint": "^8.2.6",
    "babel-preset-expo": "^4.0.0",
    "eslint": "^5.2.0",
    "eslint-plugin-import": "^2.13.0",
    "eslint-plugin-react": "^7.10.0",
    "husky": "0.15.0-rc.13",
    "jest-expo": "~28.0.0",
    "lint-staged": "^7.2.0",
    "react-native-scripts": "1.14.0",
    "react-test-renderer": "16.4.1"
  },
  "main": "./node_modules/react-native-scripts/build/bin/crna-entry.js",
  "lint-staged": {
    "src/**/*.js": [
      "git add .",
      "eslint --fix",
      "git add ."
    ]
  },
  "jest": {
    "preset": "jest-expo"
  }
}
```

P.S. @brentvatne @fson is project still maintaining?

**Comment by rayou** [comment]
same issue here

**Comment by fagerbua** [comment]
React Native's upgrade to Babel 7 broke `babel-preset-react-native-stage-0`, which is put in .babelrc when you eject from `create-react-native-app`. See skevy/babel-preset-react-native-stage-0#7 and skevy/babel-preset-react-native-stage-0#8.

Replacing "babel-preset-react-native-stage-0/decorator-support" with "babel-preset-react-native" in my project's .babelrc fixed the problem for me.

**Comment by SilencerWeb** [comment]
@fagerbua hmm, now i have another error:
![image](https://user-images.githubusercontent.com/20713191/43529668-3e519f5e-95c5-11e8-8cc9-0c0af27e7adb.png)

my .babelrc:
```js
{
  "presets": [
    "babel-preset-react-native"
  ],
  "env": {
    "development": {
      "plugins": [
        "transform-react-jsx-source"
      ]
    }
  },
  "plugins": [
    [
      "module-resolver",
      {
        "root": [
          "./src"
        ]
      }
    ]
  ]
}
```

**Comment by fagerbua** [comment]
@SilencerWeb Sorry, I didn't read the original error message carefully enough. The fix I suggested doesn't work unless you run the `eject` script. (I have edited my earlier comment to reflect this.) That's something you don't usually want to do, unless you have other compelling reasons (like needing to write native iOS or Android code in your project).

The underlying issue is that React Native 0.56 upgraded to use Babel 7, which is not backwards compatible with earlier versions. If you revert to use 0.55 (run `yarn add -D react-native@0.55` or `npm install --save-dev react-native@0.55` it should help. You may need to make sure you use version 16.3.1 of React and version 27 of Expo, as explained in [VERSIONS.md](https://github.com/react-community/create-react-native-app/blob/master/VERSIONS.md)

I believe that `create-react-native-app`'s dependency on Expo, which does not yet support React Native 0.56, makes it impossible to solve this issue without ejecting or degrading to version 0.55. See [Expo's release announcement](https://blog.expo.io/expo-sdk-v29-0-0-is-now-available-f001d77fadf) for more information.

**Comment by SilencerWeb** [comment]
@fagerbua  i see, thanks :)

but i think, i would better migrate to `react-native init` and use `react-native@0.56` which was released almost month ago but still hasn't been implemented in `create-react-native-app`, lol
